blog/_posts/2023-04-02-10.1-change-collection.md

```diff
@@ -116,26 +115,35 @@ In case something is missing, please [let us know](https://discord.gg/VA7wXVx).
+
+- {{ site.data.talent.sk }} increases {{ site.data.spell.lb }} damage by 30% for Elemental in PvP combat (was 115%).
+- Casting {{ site.data.spell.lb }} with {{ site.data.talent.sop }} causes 1 additional Elemental Overload in PvP combat (was 2).
+- {{ site.data.spell.lb }} damage increased by 53% for Elemental in PvP combat.
+- {{ site.data.talent.control_of_lava }} (PvP Talent) has been removed.
+- New PvP Talent: {{ site.data.talent.volcanic_surge }} – Increases the damage of {{ site.data.spell.lb }}, {{ site.data.spell.cl }}, and {{ site.data.spell.lvb }} by 20%. {{ site.data.talent.lvs }} instead reduces the cast time of your next {{ site.data.spell.lb }}, {{ site.data.spell.cl }}, or {{ site.data.spell.lvb }} by 25%, stacking up to 2 times.
 
 > Source: [Blue Post](https://us.forums.blizzard.com/en/wow/t/dragonflight-embers-of-neltharion-ptr-development-notes/1541379/4) (2023-03-14)
 
-- New PvP Talent: Burrow – Burrow beneath the ground, becoming unattackable, removing movement impairing effects, and increasing your movement speed by 50% for 5 seconds. When the effect ends, enemies within 6 yards are knocked in the air and take Physical damage.
-- New PvP Talent: Volcanic Surge – Increases the damage of Lightning Bolt, Chain Lightning, and Lava Burst by 20%. Lava Surge instead reduces the cast time of your next Lightning Bolt, Chain Lightning, or Lava Burst by 25%, stacking up to 2 times. Volcanic Surge now also increases the chance for Lava Surge to trigger by 100%.
+- New PvP Talent: {{ site.data.talent.burrow }} – Burrow beneath the ground, becoming unattackable, removing movement impairing effects, and increasing your movement speed by 50% for 5 seconds. When the effect ends, enemies within 6 yards are knocked in the air and take Physical damage.
+- {{ site.data.talent.volcanic_surge }} gives {{ site.data.talent.lvs }} an additional 100% chance to proc.
 
 > Source: [Blue Post](https://us.forums.blizzard.com/en/wow/t/dragonflight-embers-of-neltharion-ptr-development-notes/1541379/6) (2023-03-22)
 
-- Swelling Waves (PvP Talent) has been removed.
-- Spectral Recovery (PvP Talent) has been removed.
+- {{ site.data.talent.swelling_waves }} (PvP Talent) has been removed.
+- {{ site.data.talent.spectral_recovery }} (PvP Talent) has been removed.
 
 > Source: [Blue Post](https://us.forums.blizzard.com/en/wow/t/dragonflight-embers-of-neltharion-ptr-development-notes/1541379/7) (2023-03-29)
+
+- {{ site.data.talent.burrow }} damage increased by 200%.
+- {{ site.data.spell.lvb }} no longer consumes {{ site.data.talent.volcanic_surge }} and the cast time of {{ site.data.spell.lvb }} is no longer reduced by {{ site.data.talent.volcanic_surge }}.
+- {{ site.data.talent.volcanic_surge }} increases the damage of {{ site.data.spell.cl }} and {{ site.data.spell.lb }} by 25% (was 20%).
+- {{ site.data.talent.volcanic_surge }} increases the damage of {{ site.data.spell.lvb }} by 45% (was 20%).
```

Comment:
Do we need some sort of summary of such changes, to easily show the latest state? E.g. the most up to date tooltip/description?

Reply:
I don't think we do since the wowhead tooltip was added and it will be up to date at all time.

Reply:
I could maybe remove the tooltip on the deprecated version to avoid confusion.

Reply:
Yeah not sure how to go forward with this. Use a version you're comfortable with.